src/misc.c

```diff
@@ -26,11 +38,11 @@ u32 Crc(const void *buffer, u32 size)
    u32 crc = ~0;
    const u8  *position = buffer;
 
-   while (size--)
+   while (size--) 
       {
```

Comment:
what are these?  please don't modify whitespace in unrelated patches.

Reply:
sure. it is caused by the original PR.

Sublime keep removing tail space for me.

Reply:
I just git revert. I believe we should just do this as it is.